docs/installation.rst

```diff
@@ -6,21 +6,17 @@ User setup
 Install with pulp_installer (recommended)
 -----------------------------------------
 
-With the use of the ``pulp_installer`` installer you need to download a supportive
-role before you run installer.
-
 Only Fedora 29+ and CentOS 7 (with epel repository and python36) are supported.
 
 .. code-block:: bash
 
    git clone https://github.com/pulp/pulp_installer.git
    cd pulp_installer
```

Comment:
we should remove this and use the collection:
```
ansible-galaxy collection install pulp.pulp_installer
```
as we did here:
https://docs.pulpproject.org/en/master/nightly/installation/instructions.html#ansible-installation-recommended